backend/bang-ggood/src/test/java/com/bang_ggood/auth/config/CookieResolverTest.java

```diff
@@ -0,0 +1,43 @@
+package com.bang_ggood.auth.config;
+
+import com.bang_ggood.auth.controller.CookieProvider;
+import com.bang_ggood.global.exception.BangggoodException;
+import com.bang_ggood.global.exception.ExceptionCode;
+import jakarta.servlet.http.Cookie;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class CookieResolverTest {
+
+    @DisplayName("쿠키에서 토큰 값 조회 성공")
+    @Test
+    void extractToken() {
+        // given
+        String expectedToken = "eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiI2IiwiaWF0Ijox";
+        CookieResolver cookieResolver = new CookieResolver();
+        Cookie[] cookies = new Cookie[1];
+        cookies[0] = new Cookie(CookieProvider.TOKEN_COOKIE_NAME, expectedToken);
```

Comment:
```suggestion
        Cookie[] cookies = { new Cookie(CookieProvider.TOKEN_COOKIE_NAME, expectedToken) };

```

선언할 때 할당할 수도 있을 것 같아요!

Reply:
굿~ 👍